solutions/USACO dec15-feast-gold.mdx

```diff
@@ -0,0 +1,52 @@
+---
+id: usaco-574
+source: USACO Gold 2015 December
+title: Fruit Feast
+author: Danh Ta Chi Thanh
+---
+
+[Official Analysis (Java)](http://www.usaco.org/current/data/sol_feast_gold_dec15.html)
+
+
+<LanguageSection>
+
+<CPPSection>
+
+## C++ Implementation
+
+```cpp
+

```

Comment:
Idk whether this really matters, but it would be nice if you removed the empty lines between the opening and closing triple ticks for each solution (so that extra lines don't render). Otherwise, looks good to me

Reply:
The render seems fine, but for convenience, I will remove those empty lines.